addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -8,6 +8,8 @@
 "110010","110010","Debtors Control","asset_receivable","True"
 "110020","110020","Sundry Debtors","asset_receivable","True"
 "110030","110030","Debtors Control Account (PoS)","asset_receivable","True"
+"112006","112006","Debtors Suspense Account","asset_current","False"
+"113042","113042","Cash Control Account","asset_current","True"
```

Comment:
I am not sure what this is, but we can add it
also, please check the codes, and increment according to the account type (also group the new accounts with their type)
this chart increments codes by 10